### DISCUSSION TITLE
Alternate operation with size_maximum

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#655] [Q&A]
Hi,
i' ve define 3 alternative operations, each one with size_mininum and size_maximum; this is useful to model the oven capacity. Frepple is correctly choosing the operation depending on the size of the MO. 
The problem occurs when there is a request quantity greater than the maximum size of the bigger alternative. In this case, Frepple doesn't plan the demand. Why it doesn't create multiple manufacturing orders of this maximum size?

How can i do?

Thank you

**Comment by matteocarcano** [comment]
Sorry, is there a way to do it?

**Comment by jdetaeye** [comment]
It should split the requirement automatically over multiple MOs. I've used that feature before.   Not sure why it's not working for you...

**Comment by matteocarcano** [comment]
I've done the following setup:
Ope1 itemX size_minimum= 0 size _maximum=100 priority 1
Ope2 itemX size_minimum=101 size _maximum=300 priority 2
Ope3 itemX size_mininum= 301 size_maximum= 1000 priority 3

If the MO's quantity is bigger than 1000 the demand is not planned.

**Comment by jdetaeye** [comment]
Can you try setting the "minshipment" field on the demand to a value less than 1000? In theory it should also work with bigger values, but one never knows...
